src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -116,8 +117,11 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter {
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             const content = msg.content as any;
             if (content.event === 'stopped') {
-                this.runByLineThreadId = content.body.threadId;
-                this.runByLineSeq = content.seq;
+                if (this.isRunByLine) {
```

Comment:
Does this code work if i have a cell as follows:
```
def do_something():
    print("Hello")


do_something()
```

Now, when I'm at the last line, will I be able to step in?

Reply:
Or what if i add a breakpoint at line `print("Hello")`
& then hit `continue`, surely it shouldn't stop the run by line, but should step into that code.

Reply:
No, I don't remember how it used to work, I'll check

Reply:
Looks like run by line in the webview doesn't work on stable. I'll go for it anyway.

Reply:
The new approach covers that scenario